### DISCUSSION TITLE
Pull request default merge commit title is incorrect when repo is part of an org

### DISCUSSION THREAD

**Discussion by diamondavocado** [community/community#75827] [Pull Requests]
### Select Topic Area

Bug

### Body

Let's say you have the following:
* A GitHub organization called "MyOrg"
* ...which contains a private repo called "MyRepo"
* ...which contains some branches called "MyBranch1", "MyBranch2" etc.

When you create a pull request between branches, the default title for the merge commit is wrong.

Expected title: `Merge pull request #1 from MyRepo/MyBranch1`
Actual title: `Merge pull request #1 from MyOrg/MyBranch1`

Notice that the branch path is wrong; the repo name is missing. GitHub includes the organization name instead.

**Comment by akbaralinabiev** [comment]
Go to the GitHub repository:
Navigate to the repository where you are experiencing the issue.

Click on "Issues":
In the repository's menu, click on the "Issues" tab.

Click on "New Issue":
On the "Issues" page, click on the green "New Issue" button.

Choose a template or start a blank issue:
GitHub provides templates for various types of issues. You can choose an appropriate template if available, or start with a blank issue.

Provide a clear title:
Give your issue a clear and concise title, such as "Incorrect Default Title for Merge Commit in Pull Requests."

Provide detailed information in the body:
In the description, provide detailed information about the issue, including steps to reproduce, the expected behavior, and the actual behavior. You've already outlined this quite well in your initial message here.

Include screenshots if possible:
If applicable, include screenshots that illustrate the issue. This can help developers and maintainers understand the problem more quickly.

Assign labels and a milestone (if applicable):
GitHub allows you to assign labels and milestones to issues. Assign relevant labels, such as "bug" or "documentation," and a milestone if it fits into a specific development timeline.

Submit the issue:
Once you've filled in all the necessary information, click on the "Submit new issue" button.

**Reply by diamondavocado** [reply]
Was your answer generated by ChatGPT?

Anyway, the repository is **my** repository. Why would I submit an issue to my own repo? This is a GitHub bug.
